DB.sql

```diff
@@ -0,0 +1,41 @@
+DROP DATABASE IF EXISTS ttb;
```

Comment:
Consider using the ` delimiter around your table and column names for more clarity.